src/TodoList.jsx

```diff
@@ -1,10 +1,10 @@
 import TodoListItem from "./TodoListItem";
 
-function TodoList({ todoList }) {
+function TodoList({ todoList, onRemoveTodo }) {
     return (
         <ul>
             {todoList.map((item) => (
-                <TodoListItem key={item.id} title={item.title} />
+                <TodoListItem key={item.id} title={item.title} onRemoveTodo={() => onRemoveTodo(item.id)} />
```

Comment:
This is a minor mistake.  You should pass `id` up from `TodoListItem`.  Here you're passing an arrow function with no arguments as the `onRemoveTodo` prop, so the `id` from `TodoListItem` gets dropped.  Your code is working find for now, but it's a big opportunity for bugs.

Reply:
I'm not understanding what youre asking me? This is supposed to be for the remove button. So the id gets dropped when you click the remove button. Heres the code in the TodoListItem.jsx, where the remove button is. 

const TodoListItem = ({ title, id, onRemoveTodo }) => {
    return (
        <li>
            {title}
            <button type="button" onClick={() => onRemoveTodo(id)}>Remove</button>
        </li>
    );
}

export default TodoListItem;

Reply:
The code in `TodoListItem.jsx` is fine.  The line in `TodoList.jsx` should be:
~~~
<TodoListItem key={item.id} title={item.title} onRemoveTodo={onRemoveTodo} />
~~~
It's a subtle thing, but `onRemoveToDo` is a prop passed to the child, but its argument is being controlled by the parent.  As you had implemented it, `onRemoveToDo` is called with `item.id` regardless of what happens in the child `TodoListItem` component.  In `TodoListItem.jsx` you could have written:
~~~
<button type="button" onClick={() => onRemoveTodo("foobar")}>Remove
~~~
and it would have done the same thing.  Does that make sense?

Reply:
oh ok I get it. Thanks for helpin me understand.